config/locales/cy.yml

```diff
@@ -57,6 +61,8 @@ cy:
       govuk_date: '%-e %B %Y'
 
   time:
+    am: am
```

Comment:
Can we add a quick spec that attempts to format using these so this doesn't regress?